test/integration/054_adapter_methods_test/macros/rename_named_relation.sql

```diff
@@ -0,0 +1,6 @@
+-- Macro to rename a relation
+{% macro rename_named_relation(from_name, to_name) %}
+{%- set from_relation = adapter.get_relation(database=target.database, schema=target.schema, identifier=from_name) -%}
+{%- set to_relation = adapter.get_relation(database=target.database, schema=target.schema, identifier=to_name) -%}
```

Comment:
I think this needs to use `api.Relation.create()` rather than `adapter.get_relation()`, because if the relation doesn't exist yet, the value will be `None`.

Reply:
Oh good call, except I'm guessing you meant type = Table instead of View as the view rename isn't currently supported

Reply:
Oh, yes, that's exactly what I meant! I think it's actually got to be just the string `'table'` - `RelationType` isn't available in macros.